### ISSUE TITLE
508-defect-2 [KEYBOARD & AXE-CORE]: ARIA hidden elements must not contain focusable elements

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#26896]
# [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** _Trevor_

## User Story or Problem Statement

As a screen reader user, I might be confused if I can set focus on an A tag in the map (using my TAB key) but not hear the map or a description announced.

## Details

Axe is reporting a violation where a focusable A tag is present in a `div[aria-hidden="true"]`. This div is wrapping the map box and is likely not being picked up by our daily scan because the map is loading slower than the scan is running. Screenshot attached below.

## Dev Notes from grooming with Trevor
- [ ] we can remove aira hidden = true from map

## Acceptance Criteria

- [ ] 0 axe-core violations

## Environment

* https://www.va.gov/birmingham-health-care/locations/florence-va-clinic/

## Screenshots or Trace Logs

![Screen Shot 2021-07-01 at 4 21 42 PM](https://user-images.githubusercontent.com/934879/124191263-2ae5a580-da89-11eb-85ae-00c6902740ca.png)

**Comment by mmiddaugh** [comment]
Run axe check in local - verify it does not throw error

**Comment by mmiddaugh** [comment]
@1Copenut, when you have a moment, this one is ready for validation, please.

**Comment by 1Copenut** [comment]
@mmiddaugh I rechecked the Florence VAMC url listed above. The axe issue is no longer appearing. Moving the ticket to closed.
